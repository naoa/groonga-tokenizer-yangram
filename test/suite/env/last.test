#@omit "ommited."
#export GRN_VGRAM_ADD_ALL_EXPAND_TOKEN=1

register tokenizers/yangram

table_create vgram_words TABLE_HASH_KEY ShortText
load --table vgram_words
[
{"_key": "データ"},
{"_key": "デー"}
]
tokenize TokenYaVgramQuad "動画データス" NormalizerAuto --mode ADD
tokenize TokenYaVgramQuad "動画データス" NormalizerAuto --mode GET
