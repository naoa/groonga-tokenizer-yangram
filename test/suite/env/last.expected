register tokenizers/yangram
[[0,0.0,0.0],true]
table_create vgram_words TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
load --table vgram_words
[
{"_key": "データ"},
{"_key": "デー"}
]
[[0,0.0,0.0],2]
tokenize TokenYaVgramQuad "動画データス" NormalizerAuto --mode ADD
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "動画",
      "position": 0
    },
    {
      "value": "画デー",
      "position": 1
    },
    {
      "value": "データス",
      "position": 2
    },
    {
      "value": "ータ",
      "position": 3
    },
    {
      "value": "タス",
      "position": 4
    },
    {
      "value": "ス",
      "position": 5
    },
    {
      "value": "画デ",
      "position": 6
    },
    {
      "value": "デー",
      "position": 7
    },
    {
      "value": "データ",
      "position": 8
    }
  ]
]
tokenize TokenYaVgramQuad "動画データス" NormalizerAuto --mode GET
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "動画",
      "position": 0
    },
    {
      "value": "データス",
      "position": 2
    }
  ]
]
